sonic-slave-buster/Dockerfile.j2

```diff
@@ -341,7 +341,7 @@ RUN export VERSION=1.14.2 \
  && rm go$VERSION.linux-*.tar.gz
 
 RUN pip3 install --upgrade pip
-RUN pip2 install --upgrade pip
+RUN pip2 install --upgrade 'pip<21'
```

Comment:
not sure why need this

Reply:
In January, pip version 21 will be released, which will drop support for Python 2. Therefore, we need to ensure we install a version < 21 for Python 2 purposes.